brainscore_vision/models/resnet50_robust_l2_eps1/model.py

```diff
@@ -0,0 +1,82 @@
+from brainscore_vision.model_helpers.check_submission import check_models
+from brainscore_vision.model_helpers.activations.pytorch import PytorchWrapper
+from brainscore_vision.model_helpers.activations.pytorch import load_preprocess_images
+import torchvision
+import functools
+import torch
+import os
+from torchvision.models import resnet50
+from brainscore_vision.model_helpers.s3 import load_weight_file
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+"""
+Template module for a base model submission to brain-score
+"""
+
+
+def get_model_list():
```

Comment:
You can delete this method, specifically lines 17-24. This was an old method that Martin let us know we do not need anymore in 2.0 converted models.